src/open_samus_returns_rando/files/levels/s070_area7.lua

```diff
@@ -889,6 +896,11 @@ function s070_area7.OnStartManicMinerBotDeathCutscene()
   Game.AddEntityToUpdateInCutscene("LE_ManicMinerWall")
   Game.AddEntityToUpdateInCutscene("LE_PowerUp_Powerbomb")
   Scenario.WriteToBlackboard("manicMinerDead", "b", true)
+  if PBLauncher + Game.GetItemAmount(Game.GetPlayerName(), "ITEM_WEAPON_POWER_BOMB") > 0 then
+    Game.SetItemAmount(Game.GetPlayerName(), "ITEM_WEAPON_POWER_BOMB", 1)
+    Game.SetItemAmount(Game.GetPlayerName(), "ITEM_WEAPON_POWER_BOMB_MAX", "ITEM_POWER_BOMB_TANKS")
```

Comment:
I think there are two issues here:
1. I only used `Game.SetItemAmount` with a number as third parameter. `"ITEM_POWER_BOMB_TANKS"` is just a string.

2. You are now using `"ITEM_POWER_BOMB_TANKS"` to store `PreFightPBMax`. Which I think is a very good idea but then you need to change the if clause in line 899.
If you receive the Launcher mid fight, all of the locked `ITEM_POWER_BOMB_TANKS` will be unlocked by our Powerbomb pickup logic. In that case you don't need to anything after the fight.
The only case you need to do something is if the main item was removed pre fight and needs to be unlocked post fight.
See:

https://github.com/randovania/open-samus-returns-rando/blob/ec559eabd04df7071c602e2bba0cdf471f5125ad/src/open_samus_returns_rando/files/templates/randomizer_powerup.lua#L195-L210

Reply:
1. Good catch
2. That makes sense since the other case is already handled in our code